outlines/exploring-graphiql-the-in-browser-ide-for-graphql.md

```diff
@@ -0,0 +1,187 @@
+# Exploring GraphiQL: The In-Browser IDE for GraphQL
+
+## Introduction
+
+A. Definition and overview of GraphiQL
+
+## Why use GraphiQL?
+
+A. Advantages/benefits of using GraphiQL
+
+## Accessing GraphiQL
+
+For this guide, accessing GraphiQL can be done using any of the methods below:
+
+A. Simply head over to [https://tailcall.run/playground/](https://tailcall.run/playground/) and give your graphql server url in input. Note, make sure your graphql server allows cors for Tailcall domain
+
+B. After starting your [GraphQL server](https://tailcall.run/docs/#starting-the-graphql-server), head over to the GraphiQL playground link you get
+
+## Setting up GraphiQL
+
+A. Configuring GraphiQL
+
+B. Connecting GraphiQL to your Tailcall GraphQL server
+
+## Exploring the GraphiQL Interface
+
+A. Exploring the GraphiQL interface and understanding the functions of the various panes.
+
+B. Explain each of the main parts in the GraphiQL interface
+
+- Query Editor
+- Variables Editor
+- Response pane
+- Documentation Explorer.
+
```

Comment:
How to add headers?

Reply:
Do you mean the parts I added "head to"?

Reply:
How to add headers? in GraphiQL

Reply:
Alright, I'd add that.